openwisp_utils/admin_theme/static/admin/css/ow-dashboard.css

```diff
@@ -1,5 +1,5 @@
 #plot-container {
-  padding: 0 40px 10px 40px;
+  padding: 10px 40px;
```

Comment:
I would encourage you to experiment with other CSS units, such as `fr`, or tie it to the content's text size by using `rem` instead of hardcoding it with `px`.

Reply:
@nickberry17 Should I use it everywhere?

Reply:
To do that the style and probably the HTML would need to be refactored.  However, judicious use of the new CSS features like `grid`, `flexbox`, `clamp()` and `calc()` could probably result in a more extensible and maintainable result with far less code.  Semi-related, I am seeing `!important` sprinkled about in the other CSS files which indicates that the structure of the CSS should probably be thought about again.  Check YouTube for some[ CSS best practices 2021](https://www.youtube.com/results?search_query=css+best+practices+2021) and you will see what I mean.  Google puts out some videos, and there are others.